### PR TITLE
docs(#1794): remove unnecessary `imply`

### PR DESCRIPTION
**Pull request by JayKan**
fixing typos

### CODE REVIEW

**Comment by jayphelps** [comment]
@JayKan Just needs a rebase and we'll merge. Thanks!

File: src/Observable.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ export class Observable<T> implements Subscribable<T> {
         } else {
           // if there is NO subscription, then we're getting a nexted
           // value synchronously during subscription. We can just call it.
-          // If it errors, Observable's `subscribe` imple will ensure the
```
Comment (jayphelps) [inline src/Observable.ts]: btw I believe this was intended to be 

> Observable's subscribe _simply_ will ensure the

but it reads better without it, so 👍

**Comment by jayphelps** [comment]
@JayKan Just needs a rebase and we'll merge. Thanks!

**Comment by jayphelps** [comment]
LGTM

**Comment by JayKan** [comment]
No problem at all @jayphelps. I should have dropped by and say hi to both you and ben at the Angular meet-up event...

**Comment by jayphelps** [comment]
@JayKan indeed! Next time 🍻

**Pull request merged**